### ISSUE TITLE
GridWorld 

### ISSUE THREAD

**Issue by PengXiao9912** [Unity-Technologies/ml-agents#5219]
In example of GridWorld,i want to change the gridsize to expand the plane that it will place more obstacles ，but i have trained ,the reward is too bad!!!the reward is max_step*(the reward of each reward),that means agent cann't hit obstacles or goal.i don't know what cause it?
![image](https://user-images.githubusercontent.com/78125815/113390042-95704200-93c3-11eb-97c4-5750e5e230f7.png)

winodws 
tensorflow:2.0.4
ml-agents:0.15.0
trainer:PPO

**Comment by dongruoping** [comment]
Hi, 

There are many potential reasons that make the training not working with larger grid size. Expanding the size might make the reward more sparse and thus harder to train, and you might also need to adjust your training parameters.

In this case I don't see obvious bugs in the toolkit that caused the training to fail and it's more likely a result of the training setup and your custom environment, so I'm closing this issue here. I'd suggest you make a post on our [forum](https://forum.unity.com/forums/ml-agents.453/) which is a better place for general discussions, and put more details like how you modified the environment, what have you tried, etc.
